rxdogtag/src/test/java/com/uber/anotherpackage/RxDogTagConfigurationTest.java

```diff
@@ -213,6 +213,26 @@ public void repackagingOENIEs_disabled() {
     assertThat(cause.getStackTrace()[1].toString()).contains(RxDogTag.STACK_ELEMENT_SOURCE_HEADER);
   }
 
+  @Test
+  public void guardObserverChecks_disabled() {
```

Comment:
Can you explain how this tests guarded delegate calls? I don't see a delegate implemented here that would throw

Reply:
It actually doesn't. It's just a sanity check to make sure that even if we disable guarded delegate calls, our normal workflow works correctly. It's basically checking the `if-else` statement in the DogTag*Observers

Reply:
Let's name or comment this to make that clear

Reply:
ae2e5d5